### DISCUSSION TITLE
vips dzsave quality issue.

### DISCUSSION THREAD

**Discussion by qudratullah1061** [libvips/libvips#2191] [Q&A]
Hi,
I am using this command to convert tif to pyramidal tiled tif "vips tiffsave input_image.tif output_image.tif --tile --pyramid --compression none --tile-width 256 --tile-height 256"

To generate static tiles i am using this command.

vips dzsave input_image.tif outputfolder --layout iiif --id /test

I am serving static tiles to viewer and if user request dynamic content i am getting it from image server.

My question is: 
When i see image in opencdragon image quality is different comparitively to image getting from image server from pyramidal tif generated by libvips. You can see the difference using following links.

Image by image server using pyramidal tiff:
http://54.85.241.249/iiif/iiif/baf__b600d01f3084410eeb941cd0ed9892dfdf7a9fa9e90246eb339ebc41102bf789/full/full/0/default.jpg

Opencdragon:
http://54.85.241.249/iiif/iiif/baf__b600d01f3084410eeb941cd0ed9892dfdf7a9fa9e90246eb339ebc41102bf789

Why there is difference? How can i resolve this issue to make images consistent? Any help?

Thanks.

**Comment by jcupitt** [comment]
Hi @qudratullah1061,

I'd guess one has a profile and one does not.

dzsave strips tiles by default, otherwise you'd need a complete copy of the image metadata on each tile. You can keep the profile with:

```
vips dzsave input_image.tif outputfolder --layout iiif --id /test --no-strip
```

Or alternatively transform to sRGB before dzsave, so tiles will render without a profile:

```
vips colourspace input_image.tif outputfolder.dz[layout=iiif,id=test,no-strip] srgb
```

**Reply by qudratullah1061** [reply]
Thank you very much. Solved my problem.
